OneCommunityGlobal/HighestGoodNetworkApp
# Angelina - Add GDoc Link to Weekly Summaries Reports

1. [this icon](https://www.uplabs.com/posts/google-go-android-icon) and a link to that Google Doc for each person who has one
2. Put the icon/link to the right of their name Navigate as an Admin to Reports → Weekly Summaries Reports: I’d like the app to check for a Google Doc link on their profile (one is added for almost every person when they are added to the system) and insert 
3. On that icon, please also be sure that I can command (mac)/control (PC) + Click to open that in a new tab. It should happen automatically if it is a linked icon, but please double-check to be sure.
4. If no Google Doc is present for the user, it should instead cause a popup that says: “Uh oh, no Google Doc is present for this user! Please contact an Admin to find out why.”

### **Main Changes Explained:**

1. add google doc icon image
2. add  google doc styling in WeeklySummariesReport.css
3. add functions to FormattedReport.jsx [mainly google doc link and toast message was added]


### **How to Test:**

1. check out this branch and the backend branch that is related to this PR as well. (see link below)
2. Navigate as an Admin to Reports → Weekly Summaries Reports
3. Select the google doc icon located next to each user profile name. 
4. If they have a google doc link then a new window will open and the google doc link will appear. 
5. If they do not have a google doc link then a pop-up will appear. 

### **Additional Notes:**
This branch needs the backend branch:
https://github.com/OneCommunityGlobal/HGNRest/pull/329
